### ISSUE TITLE
Installed path isn't discovered

### ISSUE THREAD

**Issue by jdtsmith** [asmagill/hs._asm.axuielement#14]
By default `make install` places this into `~/.hammerspoon/hs/_asm/hs._asm.axuielement/` which is a malformed name.  I had to symlink that directory as `axuielement` to load.

**Comment by asmagill** [comment]
Sorry for the delay in responding, been away and trying to get back to Hammerspoon...

Technically what the Makefile is doing is looking at the directory name where the Makefile is located and using that as the last component of the path in the `~/.hammerspoon` directory...

Originally this was part of grouping of modules, so even when you cloned the repository with `git clone`, it ended up in a directory named `axuielement`, so everything worked out.  I missed this distinction when I moved it into its own repository...

The simplest fix (for the end user) would be to do something like this:

~~~sh
$ git clone https://github.com/asmagill/hs._asm.axuielement.git
$ mv hs._asm.axuielement axuielement
$ cd axuielement
...
~~~

And now, `make install` and `make uninstall` will work as expected.

Based upon some discussions at the Hammerspoon git site, this module is likely to be cleaned up and moved into the core application soon, so I'm going to update the README with what I've outlined here rather then muck with the Makefile... hopefully it won't be an issue soon.

Thanks for pointing this out -- I probably have a few other modules that should be modified in this way as well!
